### ISSUE TITLE
Bug: Performance Issues with List Attributes

### ISSUE THREAD

**Issue by GuillaumeBrn** [MISP/MISP#9164]
### Expected behavior

MISP could be returning/loading only 60 IoCs each time we access **List Attributes**, and load the next one when we access the page where they are displayed.

### Actual behavior

Each time we are loading **List Attributes** page, the loading is pretty long, and a reason could be the fact that MISP is loading all the IoCs each time (3.5 million currently on my testing environment - corresponds to 1.2GB), while it is only returning (cURL) and displaying (Web UI) 60 of them, which correspond to 40KB. I think this could be optimized (?).
The only way to reduce this was to increase my disk from 240IOPS to 500IOPS and to enable Read-only disk-caching, while I'm not sure if it's a good idea.

### Steps to reproduce

1. Import big number of IoCs
2. a. Access **List Attributes** page
    b. time curl -H "Authorization: API_KEY" -H "Accept: application/json" -H "Content-type: application/json" https://MISP-INSTANCE/attributes

### Version

2.4.171

### Operating System

Host : RH8 ; Container : CentOS 8

### Operating System version

RH8

### PHP version

7.4.30

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by srPuebla** [comment]
Yes, i have 12kk of attributes (on the table) and i am experiencing also a huge delay on each request at web page /attributes/index. I dont understand why, cause if you paginate, you should load only 60 registers on "limit". 


My Misp version es 2.4.169

**Comment by cristiancamps93** [comment]
Same happens to me. No way you can take more than 60 items per API call.

**Comment by adulau** [comment]
2.4.169 is pretty old... in any case, 2.4.192 is the way to go and includes many improvements. 

Performance issue could be also related to the MySQL configuration of your server. The settings page in MISP provides a nice way to see the optimal parameters for your MariaDB/MySQL.
